V6_MIGRATION.md

```diff
@@ -54,7 +54,7 @@ The possible values for `BTVenmoPaymentMethodUsage` include:
 * `.multiUse` - the Venmo payment will be authorized for future payments and can be vaulted.
 * `.singleUse` - the Venmo payment will be authorized for a one-time payment and cannot be vaulted.
 
-`BTVenmoClient.tokenizeVenmoAccount(with: BTVenmoRequest, completion: @escaping (BTVenmoAccountNonce?, Error?) -> Void)` has been renamed to `BTVenmoClient.tokenize(_ request: BTVenmoRequest, completion: @escaping (BTVenmoAccountNonce?, Error?) -> Void)`
+`BTVenmoClient.tokenizeVenmoAccount(with:completion:)` has been renamed to `BTVenmoClient.tokenize(_:completion:)`
```

Comment:
This makes things a lot easier to read, too! We could also audit the CHANGELOG entries for the beta cleaning these up too.

Reply:
I looked through the CHANGELOG entries and wasn't able to find anywhere where we had the other format, it looked like so far we've always used this syntax there.